.gitignore

```diff
@@ -1,2 +1,4 @@
+.vs/
 bin/
-obj/
\ No newline at end of file
+obj/
+Debug/
```

Comment:
There is no such directory. Please remove the "Debug/" - entry

Reply:
Also delete the .vs folder god damn it

Reply:
the Debug might be important for other subdirectories tho -> so `*/Debug/` will be ignored